CONTRIBUTING.md

```diff
@@ -82,15 +82,13 @@ Ready to contribute? Here's how to set up `textX` for local development.
 
         $ git clone git@github.com:your_name_here/textX.git
 
-3. Install your local copy into a virtualenv. Assuming you have
-   [virtualenv](https://virtualenv.pypa.io/en/latest) installed, this is how you
-   set up your fork for local development:
+3. Install your local copy into a virtualenv. This is how you set up your fork
+   for local development:
 
         $ cd textX/
-        $ virtualenv venv
+        $ python -m venv venv
         $ source venv/bin/activate
-        $ pip install -r requirements_dev.txt
-        $ pip install -e .
+        $ ./install-dev.sh 
```

Comment:
This gave me
```
ERROR: Could not find a version that satisfies the requirement textX-dev (from textX==2.2.0.dev0) (from versions: none)   
```
(may be ok, because I test the branch)

Reply:
I have a similar version issue while running `install-dev.sh` in textX-dev...

Reply:
After install "https://github.com/textX/textx-lang-questionnaire" I managed to run the textx-dev tests successfully. OK!

Reply:
After installing textX-dev manually, the script is working.

Reply:
Yeah, textx-lang-questionnaire is not yet deployed to PyPI, so once we get that uploaded it will be pulled in from there. Also, textX-dev is not yet deployed, so once it does you will be able to do `pip install textX[dev]` to pull in optional `dev` dependencies. 

That `[dev]` suffix is a setuptools way to specify dependencies that are optional and only needed if some particular feature is desired. That extra dependencies can be called however you like and if you want to install some of those you just use `[]` suffix with the names of optional deps separated by commas. I named these dependencies `test` and `dev` for obvious reasons :) This should be cleaner than using requirements file.